portfolio/src/main/webapp/newScript.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Adds a random fact to the page.
+ */
+
+
```

Comment:
Nit-pick: JSDoc should be directly above the function (without the extra space between) to work properly. This also makes it more clear which function the documentation is referring to.